src/conf_mode/https.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 #
-# Copyright (C) 2019-2023 VyOS maintainers and contributors
+# Copyright (C) 2019-2020 VyOS maintainers and contributors
```

Comment:
Return 2023 :)

Reply:
But I reverted the commit, that file remains unchanged. :)